lib/vsc/install/commontest.py

```diff
@@ -137,10 +137,23 @@ class CommonTest(TestCase):
     ]
 
     # Prospector commandline options (positional path is added automatically)
+    prospector_profile = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+        'etc', 'pylint_py3_checker.yaml')
+    # The prospector_profile (in theory) should be in `lib/etc` in the distribution file and
+    # in `etc` after installation. (Chosen in a way to be the same place realtively to shared_setup.py
+    if not os.path.exists(prospector_profile):
+        log.error("File prospector_profile: '%s' not found" % prospector_profile)
+        raise Exception("File prospector_profile: '%s' not found" % prospector_profile)
+
     PROSPECTOR_OPTIONS = [
-        '--strictness', 'medium',
+        '--profile', prospector_profile,
```

Comment:
has this been tested on the current python 2 tests? Do the ones we defined as 'mandatory' (in https://github.com/hpcugent/vsc-install/blob/481f8e50921d35cfb12995999297a1cb49e46f58/lib/vsc/install/commontest.py#L100) still work/fail?

Reply:
@wdpypere using `pylint --py3k` indeed suppresses all messages but the python 3 compatibility ones, however, `pylint --enable=python3` does not suppress any message and still checks python3 compatibility (I have used this method). 

On the other hand, the default strictness for `prospector`  (medium, and this was explicitly requested in the command line arguments in the past, and in the current version prospector_profile inherits the medium strictness) suppresses some messages that are in the WHITELIST, like `trailing-whitespace` and `bad-whitespace`,  Some other tests that checked by prospector at medium strictness level, like `unused-import` still breaks the test.

Reply:
can you add unittests for this? (i know for a fact that trailing whitespace fails the unittest)

Reply:
@stdweird Maybe I am using some other versions, but for me. trailing white spaces does not fail the test:
```
hajgato@DESKTOP-ECJFGEJ:~/work/vsc-install$ python setup.py test
....
Ran 17 tests in 6.918s
OK
hajgato@DESKTOP-ECJFGEJ:~/work/vsc-install$ git status
On branch master
Your branch is up to date with 'origin/master'.
Changes not staged for commit:
(use "git add <file>..." to update what will be committed)
(use "git checkout -- <file>..." to discard changes in working directory)
modified:   lib/vsc/install/shared_setup.py
no changes added to commit (use "git add" and/or "git commit -a")
hajgato@DESKTOP-ECJFGEJ:~/work/vsc-install$ git diff
diff --git a/lib/vsc/install/shared_setup.py b/lib/vsc/install/shared_setup.py
index a64f852..733ae7f 100644
--- a/lib/vsc/install/shared_setup.py
+++ b/lib/vsc/install/shared_setup.py
@@ -32,7 +32,7 @@ Shared module for vsc software setup
"""
from __future__ import print_function
-import sys
+import sys         
if sys.version_info < (3, 0):
import __builtin__
hajgato@DESKTOP-ECJFGEJ:~/work/vsc-install$ pylint setup.py |grep -i  white
No config file found, using default configuration
C: 35, 0: Trailing whitespace (trailing-whitespace)
```